charts/adot-exporter-for-eks-on-ec2/templates/adot-collector/configmap.yaml

```diff
@@ -9,39 +9,60 @@ metadata:
     extensions:
       health_check: {{ .Values.adotCollector.daemonSet.extensions.healthCheck }}
     receivers:
       awscontainerinsightreceiver:
-        collection_interval: {{ .Values.adotCollector.daemonSet.receivers.collectionInterval }}
-        container_orchestrator: {{ .Values.adotCollector.daemonSet.receivers.containerOrchestrator }}
-        add_service_as_attribute: {{ .Values.adotCollector.daemonSet.receivers.addServiceAsAttribute }}
-        prefer_full_pod_name: {{ .Values.adotCollector.daemonSet.receivers.preferFullPodName }}
+        collection_interval: {{ .Values.adotCollector.daemonSet.cwreceivers.collectionInterval }}
+        container_orchestrator: {{ .Values.adotCollector.daemonSet.cwreceivers.containerOrchestrator }}
+        add_service_as_attribute: {{ .Values.adotCollector.daemonSet.cwreceivers.addServiceAsAttribute }}
+        prefer_full_pod_name: {{ .Values.adotCollector.daemonSet.cwreceivers.preferFullPodName }}
+      prometheus:
+        config:
+          global:
+            scrape_interval: {{ .Values.adotCollector.daemonSet.ampreceivers.scrapeInterval }}
+            scrape_timeout: {{ .Values.adotCollector.daemonSet.ampreceivers.scrapeTimeout }}
+          scrape_configs: {{ .Values.adotCollector.daemonSet.ampreceivers.scrapeConfigs | nindent 10 }}
     processors:
       batch/metrics:
         timeout: {{ .Values.adotCollector.daemonSet.processors.timeout }}
     exporters:
       awsemf:
-        namespace: {{ .Values.adotCollector.daemonSet.exporters.namespace }}
+        namespace: {{ .Values.adotCollector.daemonSet.cwexporters.namespace }}
         log_group_name: '/aws/containerinsights/{{ .Values.clusterName }}/performance'
-        log_stream_name: {{ .Values.adotCollector.daemonSet.exporters.logStreamName }}
+        log_stream_name: {{ .Values.adotCollector.daemonSet.cwexporters.logStreamName }}
         resource_to_telemetry_conversion:
-          enabled: {{ .Values.adotCollector.daemonSet.exporters.enabled }}
-        dimension_rollup_option: {{ .Values.adotCollector.daemonSet.exporters.dimensionRollupOption }}
-        parse_json_encoded_attr_values: {{- range .Values.adotCollector.daemonSet.exporters.parseJsonEncodedAttrValues }}
+          enabled: {{ .Values.adotCollector.daemonSet.cwexporters.enabled }}
+        dimension_rollup_option: {{ .Values.adotCollector.daemonSet.cwexporters.dimensionRollupOption }}
+        parse_json_encoded_attr_values: {{- range .Values.adotCollector.daemonSet.cwexporters.parseJsonEncodedAttrValues }}
         - {{.}}{{- end }}
         metric_declarations:
           {{ .Values.adotCollector.daemonSet.metricDeclarations | nindent 10 }}
+      awsprometheusremotewrite:
+        namespace: {{ .Values.adotCollector.daemonSet.ampexporters.namespace }}
+        endpoint: {{ .Values.adotCollector.daemonSet.ampexporters.endpoint }}
+        # resource_to_telemetry_conversion: 
+        #   enabled: {{ .Values.adotCollector.daemonSet.ampexporters.resourcetotel }}
```

Comment:
Why is is this commented out?

Reply:
I think adding the relabling code block resolved the out of sample error. So, this code block maybe isn't required? But I think we should keep it anyway to add info when the metrics are sent.

Reply:
Uncomment this block and set the default value to `false` if it isn't necessary for the default configuration to work.